src/index.html

```diff
@@ -8,7 +8,48 @@
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="Home">
+    <header class="header">
+      <a class="logo-href" href="#Home">
+        <img src="./images/logo.png" alt="Moyo-logo"
+        class="logo__img">
```

Comment:
Pay attention on code style
```suggestion
        <img
          src="./images/logo.png" 
          alt="Moyo-logo" 
          class="logo__img"
        >
```